cli/Valet/PhpFpm.php

```diff
@@ -120,4 +118,53 @@ function fpmConfigPath()
             ? '/usr/local/etc/php/5.6/php-fpm.conf'
             : "/usr/local/etc/php/${versionNormalized}/php-fpm.d/www.conf";
     }
+
+    /**
+     * Only stop running php services
+     */
+    function stopRunning()
+    {
+        $this->brew->stopService(
+            $this->brew->getRunningServices()
+                ->filter(function ($service) {
+                    return substr($service, 0, 3) === 'php';
+                })
+                ->all()
+        );
+    }
+
+    /**
+     * Use a specific version of php
+     *
+     * @param $version
+     * @return string
+     */
+    function useVersion($version)
+    {
+        // If passed php7.2 or php72 formats, convert to php@7.2 format:
+        $version = preg_replace('/(php)([0-9+])(?:.)?([0-9+])/i', '$1@$2.$3', $version);
+
+        if (!$this->brew->supportedPhpVersions()->contains($version)) {
+            throw new DomainException(
+                sprintf('Valet doesnn\'t support PHP version: %s', $version)
```

Comment:
Typo in "doesn't"
but also maybe we should suggest one?

```php
sprintf("Valet doesn't support PHP versions: %s (try something like 'php71' instead)"
```

And that does also make me curious whether we should consider using "7.1" instead of "php7.1" or whatever as our syntax? `valet use 7.1`